packages/hebao_v1/contracts/modules/base/BaseModule.sol

```diff
@@ -129,7 +129,15 @@ abstract contract BaseModule is ReentrancyGuard, Module
         internal
         returns (bytes memory)
     {
-        return Wallet(wallet).transact(uint8(1), to, value, data);
+        bytes memory callData = abi.encodeWithSelector(
```

Comment:
Shall we also use `fastCallAndVerify` in `transactDelegateCall` and `transactStaticCall`>

Reply:
Or should we change `memory` to `calldata` as in `transactDelegateCall` without the function's body change?

Reply:
Sometimes `transactCall` is used with a `calldata` data from the caller; sometimes the data is assembled in memory.